src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByDecreasingQuantity.java

```diff
@@ -6,6 +6,12 @@
 
 public class ItemComparatorByDecreasingQuantity implements Comparator<Item> {
 
+    /**
+     * Compares between 2 item by seeing which item has the lower quantity.
```

Comment:
I think you need a space between the params.

Reply:
noted!